plugin/rasterizing/rasterizing/src/test/java/org/geomajas/plugin/rasterizing/tms/TmsServiceTest.java

```diff
@@ -0,0 +1,108 @@
+				"<TileMapService>\n" + 
+				"    <Title>Geomajas Tile Map Service</Title>\n" + 
+				"    <Abstract>Tile Map Service publication of the Geomajas vector layers</Abstract>\n" + 
+				"    <TileMaps>\n" + 
+				"        <TileMap href=\"http://test/tms/1.0.0/beans@EPSG:3857/beansStyleInfo\" profile=\"global-mercator\" srs=\"EPSG:3857\" title=\"beans\"/>\n" + 
+				"        <TileMap href=\"http://test/tms/1.0.0/beans@EPSG:4326/beansStyleInfo\" profile=\"global-geodetic\" srs=\"EPSG:4326\" title=\"beans\"/>\n" + 
+				"        <TileMap href=\"http://test/tms/1.0.0/beans@EPSG:4326/beansStyleInfo\" profile=\"local\" srs=\"EPSG:4326\" title=\"beans\"/>\n" + 
+				"        <TileMap href=\"http://test/tms/1.0.0/lotsObeans@EPSG:3857/beansStyleInfo\" profile=\"global-mercator\" srs=\"EPSG:3857\" title=\"lotsObeans\"/>\n" + 
+				"        <TileMap href=\"http://test/tms/1.0.0/lotsObeans@EPSG:4326/beansStyleInfo\" profile=\"global-geodetic\" srs=\"EPSG:4326\" title=\"lotsObeans\"/>\n" + 
+				"        <TileMap href=\"http://test/tms/1.0.0/lotsObeans@EPSG:4326/beansStyleInfo\" profile=\"local\" srs=\"EPSG:4326\" title=\"lotsObeans\"/>\n" + 
+				"        <TileMap href=\"http://test/tms/1.0.0/layerBeansMultiLine@EPSG:3857/layerBeansMultiLineStyleInfo\" profile=\"global-mercator\" srs=\"EPSG:3857\" title=\"layerBeansMultiLine\"/>\n" + 
+				"        <TileMap href=\"http://test/tms/1.0.0/layerBeansMultiLine@EPSG:4326/layerBeansMultiLineStyleInfo\" profile=\"global-geodetic\" srs=\"EPSG:4326\" title=\"layerBeansMultiLine\"/>\n" + 
+				"        <TileMap href=\"http://test/tms/1.0.0/layerBeansMultiLine@EPSG:4326/layerBeansMultiLineStyleInfo\" profile=\"local\" srs=\"EPSG:4326\" title=\"layerBeansMultiLine\"/>\n" + 
+				"        <TileMap href=\"http://test/tms/1.0.0/layerBeansMultiPolygon@EPSG:3857/layerBeansMultiPolygonStyleInfo\" profile=\"global-mercator\" srs=\"EPSG:3857\" title=\"layerBeansMultiPolygon\"/>\n" + 
+				"        <TileMap href=\"http://test/tms/1.0.0/layerBeansMultiPolygon@EPSG:4326/layerBeansMultiPolygonStyleInfo\" profile=\"global-geodetic\" srs=\"EPSG:4326\" title=\"layerBeansMultiPolygon\"/>\n" + 
+				"        <TileMap href=\"http://test/tms/1.0.0/layerBeansMultiPolygon@EPSG:4326/layerBeansMultiPolygonStyleInfo\" profile=\"local\" srs=\"EPSG:4326\" title=\"layerBeansMultiPolygon\"/>\n" + 
+				"        <TileMap href=\"http://test/tms/1.0.0/layerBeansPoint@EPSG:3857/layerBeansPointStyleInfo\" profile=\"global-mercator\" srs=\"EPSG:3857\" title=\"layerBeansPoint\"/>\n" + 
+				"        <TileMap href=\"http://test/tms/1.0.0/layerBeansPoint@EPSG:4326/layerBeansPointStyleInfo\" profile=\"global-geodetic\" srs=\"EPSG:4326\" title=\"layerBeansPoint\"/>\n" + 
+				"        <TileMap href=\"http://test/tms/1.0.0/layerBeansPoint@EPSG:4326/layerBeansPointStyleInfo\" profile=\"local\" srs=\"EPSG:4326\" title=\"layerBeansPoint\"/>\n" + 
+				"    </TileMaps>\n" + 
+				"</TileMapService>\n" + 
+				"", sw.toString());
+	}
```

Comment:
Can expected xml code be set in a separate xml file, or using CDATA? DOn't know if it is possible, but it seems better than String on different lines.